### DISCUSSION TITLE
question regarding the font size of legends in SED plots

### DISCUSSION THREAD

**Discussion by tanofspace** [mboquien/cigale#48] [Q&A]
Is there a way to modify the font size of the legends in the SED plots generated? They are on the smaller end and it makes them quite hard to read without zooming in.

**Comment by mboquien** [comment]
The easiest would be to edit the source code of `pcigale-plots sed`. It is in `pcigale_plots/plot_types/sed/__init__.py`.

**Reply by tanofspace** [reply]
I see, thank you!
